boards/opentitan/README.md

```diff
@@ -0,0 +1,27 @@
+OpenTitan RISC-V Board
+=================
+
+- https://opentitan.org/
+
+OpenTitan is the first open source project building a transparent, high-quality reference design and integration guidelines for silicon root of trust (RoT) chips.
+
```

Comment:
This should explain how to get this board.

Reply:
Fixed in the next version